### PR TITLE
Add icons to webviews

### LINKED ISSUE #757: Use vscode icons for webview.iconPath
**Is your feature request related to a problem? Please describe.**
Currently, all webviews contributed by the extension use the default webview icon, i.e.

image

I think it would be useful to the extension's UX if we changed these to icons that better explain what the panel is. I have put a few icons together for an example:

image

**Additional context**
webview.iconPath currently only supports uris, so can't use codicons without downloading them from the [vscode icon repo](https://github.com/microsoft/vscode-icons)

### PR DESCRIPTION
**Pull request by ElianHugh**
# What problem did you solve?
Resolves #757. This adds icons to the webviews contributed by VSCode-R. At present, VSC-R webviews use the default webview panel icon. I think it would be useful if we changed these to icons that better explain what the panel is at a glance.

As `webview.iconPath` currently requires a URI, we can't use the `new ThemeIcon()` call, so icons have been obtained from the vscode icon repo. This PR adds the following icons:

- globe.svg (used by viewer and browser)
- graph.svg (used by plot viewer)
- open-preview.svg (used by data view)
- preview.svg (used by rmd preview)
- help.svg (used by help viewer)

## Screenshot

image

### CODE REVIEW

**Review by renkun-ken — APPROVED** [review]
Works nicely!

LGTM

File: src/session.ts [hunk 1]
```diff
@@ -237,6 +237,7 @@ export async function showBrowser(url: string, title: string, viewer: string | b
             }
             void commands.executeCommand('setContext', 'r.browser.active', false);
         });
+        panel.iconPath = new UriIcon('globe');
```
Comment (renkun-ken) [inline src/session.ts:240]: Why do I have

image

all these complaints from the eslint?

Comment (ManuelHentschel) [reply src/session.ts:240]: I do not get these.
Sometimes I get random `any type` warnings after e.g. switching between Git branches, and reloading the window usually resolves these.

Comment (renkun-ken) [reply src/session.ts:240]: Yes, thanks!

**Pull request merged**